Microsoft Defender for Endpoint deprecation (#37727)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/Microsoft365DefenderEventCollector/README.md

Microsoft Defender for Endpoint Alerts integration for Cortex XSIAM (Deprecated).

## Deprecation Announcement
Following [this](https://learn.microsoft.com/en-us/defender-endpoint/configure-siem) announcement by Microsoft about migrating from the deprecated SIEM API to the Graph API, this Event Collector is now deprecated.

### Replacement Option:

> Use the XSIAM `Office 365` Data Source, select `Microsoft Graph API` -> `Alerts`, and choose the `Use Microsoft Graph API V2` option.

Reviewer: ```suggestion
In XSIAM `Office 365` Data Source, click `Microsoft Graph API` > `Alerts`, and select `Use Microsoft Graph API V2`.
```
